File: TrabajoI.C/JS/interfaz/ClaseE.js
class E {
    constructor(precio) {
        this.precio = precio;
        console.log(`Constructor con parámetros en E → precio: ${precio}`);

        implementarInterfaz(this, InterfazOperable);
    }

    operar() {
        console.log("Operando en E");
    }
}

File: TrabajoI.C/C#/clase/ClaseE.cs
public class E : Figura
{
    private double precio;

    public E(double precio)
    {
        this.precio = precio;
        Console.WriteLine($"Constructor con parámetros en E: {precio}");
    }

    public override void Dibujar()
    {
        Console.WriteLine("Dibujando E");
    }
}
